internal/crypto/envelope/enveloper.go

```diff
@@ -0,0 +1,11 @@
+package envelope
+
+import "io"
+
+//go:generate mockery --name Enveloper --with-expecter=true
```

Comment:
There is no standart mock gen in project.
Should we make an issue in backlog for that?

Reply:
Yes, current zoo of mock generators is confusing and needs to be fixed eventually. Would appreciate an issue addressing this.